src/crates.rs

```diff
@@ -77,6 +80,12 @@ pub fn pre_fetch(config: &Config) {
     let config = config.clone();
     if let Some(_) = config.prefetch_path {
         let prefetch_path = config.prefetch_path.clone().unwrap();
+        let prefetch_parts: Vec<&str> = prefetch_path.split(".").collect();
+        let prefetch_ext = prefetch_parts[prefetch_parts.len() - 1];
```

Comment:
I would rather see this use [Path::extension](https://doc.rust-lang.org/std/path/struct.Path.html#method.extension) rather than manually splitting off the bit after the '.'

Reply:
Done.